src/vma/proto/dst_entry_tcp.cpp

```diff
@@ -174,6 +174,7 @@ ssize_t dst_entry_tcp::fast_send(const struct iovec* p_iov, const ssize_t sz_iov
 
 #ifndef __COVERITY__
         struct tcphdr* p_tcp_h = (struct tcphdr*)(((uint8_t*)(&(p_pkt->hdr.m_ip_hdr))+sizeof(p_pkt->hdr.m_ip_hdr)));
+        p_tcp_h = p_tcp_h; /* to supress warning in case VMA_OPTIMIZE_LOG */
```

Comment:
In order to keep consistency with other parts of the code please consider using:
```code
NOT_IN_USE(p_tcp_h); /* to suppress warning in case VMA_OPTIMIZE_LOG */
``` 
instead of
```code2
 p_tcp_h = p_tcp_h; /* to suppress warning in case VMA_OPTIMIZE_LOG */
```